addons/website_blog/static/src/js/website_blog.inline.discussion.js

```diff
@@ -36,12 +36,13 @@ var InlineDiscussion = core.Class.extend({
         self.discussions_handler(self.settings.content);
 
         // Hide the discussion.
-        $('html').click(function(event) {
-            if($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
+        $('html').click(function (event) {
+            if ($(event.target).parents('#discussions_wrapper, .main-discussion-link-wrp').length === 0) {
                 self.hide_discussion();
             }
-            if(!$(event.target).hasClass('discussion-link') && !$(event.target).parents('.popover').length){
-                if($('.move_discuss').length){
+            if ((!$(event.target).hasClass('discussion-link') || ($(event.target).hasClass('discussion-link') && $(event.target).hasClass('active'))) && !$(event.target).parents('.popover').length) {
```

Comment:
if ((!$(event.target).hasClass('discussion-link') || ($(event.target).is('.discussion-link.active'))) && !$(event.target).closest('.popover').length) {

Use 'is', it's shorter
Use closest to get the first parent who match (and not get all parents and filter)

thanks